modules/go/01_mod.mk

```diff
@@ -39,3 +45,60 @@ verify-govulncheck: | $(NEEDS_GOVULNCHECK)
 				popd >/dev/null; \
 				echo ""; \
 			done
+
+ifdef golangci_lint_config
```

Comment:
review note: This is the most important part of the PR. golangci-lint won't be enabled unilaterally - it'll only be enabled if we do another step after merging the makefile-modules upgrade.